alphadia/workflow/peptidecentric/peptidecentric.py

```diff
@@ -159,6 +154,7 @@ def __init__(
 
         self._extraction_handler: ExtractionHandler | None = None
         self._recalibration_handler: RecalibrationHandler | None = None
+        self._requantification_handler: RequantificationHandler | None = None
```

Comment:
We are changing the semantics of the handlers here a bit.

In `PeptideCentric` there are certain high level steps which are actually called like a pipeline if needed:

`PeptideCentric.search_parameter_optimization`
`PeptideCentric.extraction`
`PeptideCentric.requantify`
`PeptideCentric.requantify_fragments`

The requantification handler is now very close to requantify. I think we should make this hierarchy more clear.